visit-dav/visit
# Changed vertical scroll mode for plot list to ScrollPerPixel and single step

Resolves #3781. Changed vertical scroll mode for plot list to ScrollPerPixel and to use a single step so the plot list is not cut off.

### How Has This Been Tested?

1. Open a database
2. Add a pseudocolor plot and add several reflect operators.
3. expand the plot
4. Shrink plot list to the minimum size.
5. Vertical scroll bar should appear that allows you to scroll to the bottom of the plot list

### Checklist:

- [x] I have assigned reviewers (see [VisIt's PR procedures](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers) for more information).
